<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Poffer Game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>50</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>50</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_loginTitle">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Poffer Game Login</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameLineEdit">
          <property name="placeholderText">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="signUpButton">
          <property name="text">
           <string>Sign Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="forgotPasswordButton">
          <property name="text">
           <string>Forgot Password?</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="signUpPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_signUpTitle">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Create New Account</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="signUpNameLineEdit">
          <property name="placeholderText">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="signUpLastNameLineEdit">
          <property name="placeholderText">
           <string>Last Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="signUpPhoneNumberLineEdit">
          <property name="placeholderText">
           <string>Phone Number (e.g., 09xxxxxxxxx)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="signUpEmailLineEdit">
          <property name="placeholderText">
           <string>Email Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="signUpUsernameLineEdit">
          <property name="placeholderText">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="signUpPasswordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password (min 8 characters)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="registerButton">
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backToLoginButton">
          <property name="text">
           <string>Back to Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mainMenuPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>50</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <property name="bottomMargin">
         <number>50</number>
        </property>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_mainMenuTitle">
          <property name="font">
           <font>
            <pointsize>26</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Poffer Game</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_welcomeUser">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Welcome, Player!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startGameButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Start Game</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editProfileButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Edit Profile</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gameHistoryButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Game History</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="logoutButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Logout</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="editProfilePage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_editProfileTitle">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Edit Profile</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_name">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="editNameLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_lastName">
            <property name="text">
             <string>Last Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="editLastNameLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_phoneNumber">
            <property name="text">
             <string>Phone Number:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="editPhoneNumberLineEdit">
            <property name="placeholderText">
             <string>e.g., 09xxxxxxxxx</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_email">
            <property name="text">
             <string>Email Address:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="editEmailLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_username">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="editUsernameLineEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_password">
            <property name="text">
             <string>New Password:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="editPasswordLineEdit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>min 8 characters</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_confirmPassword">
            <property name="text">
             <string>Confirm Password:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="editConfirmPasswordLineEdit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="saveProfileButton">
            <property name="text">
             <string>Save Changes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelProfileButton">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gameHistoryPage">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_historyTitle">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Game History</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="gameHistoryTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Your past game history will appear here (max 3 games).</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backToMainMenuFromHistoryButton">
          <property name="text">
           <string>Back to Main Menu</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gamePage">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_gameTitle">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Poffer Game in Progress</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_opponentHand">
          <property name="text">
           <string>Opponent's Hand: [Cards placeholder]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_playerHand">
          <property name="text">
           <string>Your Hand: [Cards placeholder]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectCardButton">
          <property name="text">
           <string>Select Card</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="requestExchangeButton">
          <property name="text">
           <string>Request Card Exchange</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopResumeButton">
          <property name="text">
           <string>Stop/Resume Game</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="gameLogTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Game log and chat messages will appear here...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitGameButton">
          <property name="text">
           <string>Exit Game</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
